dipy/reconst/csdeconv.py

```diff
@@ -118,18 +120,20 @@ def __init__(self, gtab, response, reg_sphere=None, sh_order=8, lambda_=1,
         # scale lambda_ to account for differences in the number of
         # SH coefficients and number of mapped directions
         # This is exactly what is done in [4]_
-        self.lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
-                        (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.))
-                       )
+
+        lambda_ = (lambda_  * self.R.shape[0] * r_rh[0] /
+                   (np.sqrt(self.B_reg.shape[0]) * np.sqrt(362.)))
```

Comment:
Oh, so that would be in agreement with @arokem previous comment then. Which would also mean that the sqrt(362) is just a hardcoded value without any underlying meaning either.